src/config.ts

```diff
@@ -189,7 +191,10 @@ export class CacheConfig {
         }
       }
 
-      const cargo_locks = sort_and_uniq(await globFiles(`${root}/**/Cargo.lock`));
+      const cargo_locks = sort_and_uniq(workspaceMembers
+        .map(member => path.join(member.path, "Cargo.lock"))
+        .filter(fs.existsSync)
+      );
```

Comment:
I’m wondering if this makes sense at all, since a workspace by definition only has one `Cargo.lock` file containing everything from all the members.

Reply:
ah right, changed it to only check for a single `Cargo.lock` under each workspace root